pkg/lockfile/parse-apk-installed_test.go

```diff
@@ -0,0 +1,68 @@
+package lockfile_test
+
+import (
+	"github.com/google/osv-scanner/pkg/lockfile"
+	"testing"
+)
+
+func TestParseApkInstalled_Empty(t *testing.T) {
+	t.Parallel()
+
+	packages, err := lockfile.ParseApkInstalled("fixtures/apk/installed_empty")
```

Comment:
nit: while `installed` isn't technically the file extension, I think it would be better to name the fixtures with `_installed` at the end, as that's more consistent with the others having their file extension at the end